### PR TITLE
[sdk] Throw valid auction errors as-is

### PR DESCRIPTION
**Pull request by jakekidd**
## The Problem

We were snuffing out `NoBids` errors with `UnknownAuctionError`

## The Solution

- Added property to `AuctionError` "isAuctionError" (equal to true)
- Wrapped the whole segment of the `getTransferQuote` method in a try/catch, instead of multiple try/catches
- When we catch an error, we check `error.isAuctionError` to see if we should throw the error as-is.

## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [x] Update documentation if needed.
- [x] Update CHANGELOG.md.

### CODE REVIEW

File: packages/sdk/src/sdkBase.ts [hunk 1]
```diff
@@ -788,25 +791,28 @@ export class NxtpSdkBase {
           return data;
         }),
       );
+
+      const validBids = receivedBids.filter((x) => typeof x !== "string") as AuctionResponse[];
+      const invalidBids = receivedBids.filter((x) => typeof x === "string") as string[];
+
+      if (validBids.length === 0) {
+        throw new NoValidBids(transactionId, payload, invalidBids.join(","), receivedBids);
```
Comment (rhlsthrm) [inline packages/sdk/src/sdkBase.ts:799]: This error also needs to be `isAuctionError = true` right?

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/sdk/src/sdkBase.ts (hunk 1 above)
Comment (jakekidd) [reply packages/sdk/src/sdkBase.ts:799]: it should inherit that property

**Review by rhlsthrm — APPROVED** [review]

**Comment by rhlsthrm** [comment]
Pls fix conflicts.

**Review by LayneHaber — CHANGES_REQUESTED** [review]
Looks good, but needs unit tests

**Comment by sanchaymittal** [comment]
closing this pr, because of inactivity. Please reopen it if necessary.

**Pull request closed without merging**